kyuubi-hive-beeline/src/main/resources/BeeLine.properties

```
@@ -154,73 +154,80 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
+\  --numberFormat=[pattern]        Format numbers using DecimalFormat pattern.\n\
+\  --force=[true|false]            Continue running script even after errors.\n\
+\  --maxWidth=MAXWIDTH             The maximum width of the terminal.\n\
+\  --maxColumnWidth=MAXCOLWIDTH    The maximum width to use when displaying columns.\n\
+\  --silent=[true|false]           Be more silent.\n\
+\  --autosave=[true|false]         Automatically save preferences.\n\
+\  --outputformat=<format mode>    Format mode for result display.\n\
+\                                  The available options ars [table|vertical|csv2|tsv2|dsv|csv|tsv|json|jsonfile]. \n\
+\                                  Note that csv, and tsv are deprecated, use csv2, tsv2 instead.\n\n\
+\  --incremental=[true|false]      Defaults to false. When set to false, the entire result set\n\
+\                                  is fetched and buffered before being displayed, yielding optimal\n\
+\                                  display column sizing. When set to true, result rows are displayed\n\
+\                                  immediately as they are fetched, yielding lower latency and\n\
+\                                  memory usage at the price of extra display column padding.\n\
+\                                  Setting --incremental=true is recommended if you encounter an OutOfMemory\n\
+\                                  on the client side (due to the fetched result set size being large).\n\
+\                                  Only applicable if --outputformat=table.\n\n\
+\  --incrementalBufferRows=NUMROWS The number of rows to buffer when printing rows on stdout,\n\
+\                                  defaults to 1000; only applicable if --incremental=true\n\
+\                                  and --outputformat=table.\n\n\
+\  --truncateTable=[true|false]    Truncate table column when it exceeds length.\n\
+\  --delimiterForDSV=DELIMITER     Specify the delimiter for delimiter-separated values output format (default: |).\n\
+\  --isolation=LEVEL               Set the transaction isolation level.\n\
+\  --nullemptystring=[true|false]  Set to true to get historic behavior of printing null as empty string.\n\
+\  --maxHistoryRows=MAXHISTORYROWS The maximum number of rows to store beeline history.\n\
+\  --delimiter=DELIMITER           Set the query delimiter; multi-char delimiters are allowed, but quotation\n\
+\                                  marks, slashes, and -- are not allowed (default: ;).\n\n\
+\  --convertBinaryArrayToString=[true|false]\n\
+\                                  Display binary column data as string or as byte array.\n\n\
+\  --python-mode                   Execute python code/script.\n\
+\  -h, --help                      Display this message.\n\
+\n\
+Examples:\n\
+\  1. Connect using simple authentication to Kyuubi Server on localhost:10009.\n\
+\  $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 <username> <password>\n\n\
```

same as LDAP, it does not work too. let's replace it with the following example
```suggestion
\  $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 -n username\n\n\
```